pages/common/cpdf.md

```diff
@@ -0,0 +1,36 @@
+# cpdf
+
+> CLI to manipulate existing PDF files in a variety of ways.
+> Homepage: <https://www.coherentpdf.com/cpdfmanual/cpdfmanual.html>.
```

Comment:
After review it again to translate it to Spanish, I think the most consistent way to put an URL for the command is "More information". Although I haven't read a guideline or a rule about it, all the pages I remember to read have "More information". Should we unify the line that provides the URL?

Reply:
Yes, that's right. We should use "More information"